tasks-by-users/AbbasShaikh0304/10-Product Card UI Design/style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  margin: 0%;
+  padding: 0%;
+  font-family: "Poppins", sans-serif;
+}
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background: #131313;
+}
+.main {
+  position: relative;
+}
+.main .card {
+  position: relative;
+  width: 320px;
+  height: 450px;
+  background: #232323;
+  border-radius: 20px;
+  overflow: hidden;
+}
+.main .card::before {
+  content: "";
+  position: absolute;
+  top: 0%;
+  left: 0%;
+  width: 100%;
+  height: 100%;
+  background: #9bdc28;
+  clip-path: circle(150px at 80% 20%);
+  transition: 0.5s ease-in-out;
+}
+.main .card:hover::before {
+  clip-path: circle(300px at 80% -20%);
+}
+.main .card::after {
+  content: "Nike";
+  position: absolute;
+  top: 30%;
+  left: -20%;
+  font-size: 12em;
+  font-weight: 800;
+  font-style: italic;
+  color: rgba(255, 255, 255, 0.04);
+}
+.main .card .imageBx {
+  position: absolute;
+  top: 30%;
+  transform: translate(0%);
+  z-index: 10000;
+  width: 100%;
+  height: 220px;
+  transition: 0.5s;
+}
+.main .card:hover .imageBx {
+  top: 0%;
+  transform: translate(0%);
+}
+.main .card .imageBx img {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) rotate(-25deg);
+  width: 270px;
+}
+.main .card .contentBx {
+  position: absolute;
+  bottom: 0%;
+  width: 100%;
+  height: 100px;
+  text-align: center;
+  transition: 1s;
+  z-index: 10;
+}
+.main .card:hover .contentBx {
+  height: 210px;
+}
+.main .card .contentBx h2 {
+  position: relative;
+  font-weight: 600;
+  letter-spacing: 1px;
+  color: #fff;
+}
+.main .card .contentBx .size,
+.main .card .contentBx .color {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 8px 20px;
+  transition: 0.5s;
+  opacity: 0;
+  visibility: hidden;
+}
+.main .card:hover .contentBx .size {
+  opacity: 1;
+  visibility: visible;
+  transition-delay: 0.6s;
+}
+.main .card:hover .contentBx .color {
```

Comment:
Use multiple selector here `,`

Reply:
I didn't intended to select all those classes, i used descended selector there. why would i use multiple selector when i dont want to select those classes as well?